client/src/Component/homepage/homepage.js

```diff
@@ -0,0 +1,116 @@
+import React,{useState,useEffect} from "react";
```

Comment:
Why are you create a new Homepage component? You can refer [here](https://github.com/purnima143/Kurakoo/blob/master/client/src/components/homepage/homepage.js)